bower.json

```diff
@@ -22,34 +22,30 @@
   ],
   "dependencies": {
     "jquery": "~2.2.1",
-    "angular": "~1.4.0",
-    "angular-bootstrap": "0.13.4",
+    "angular": "~1.6.0",
+    "angular-messages": "~1.6.0",
+    "angular-bootstrap": "~2.3.0",
     "angular-ui-sortable": "~0.13.4",
     "angular-xeditable": "~0.1.9",
+    "angular-toastr": "^1.7.0",
     "bootstrap": "~3.3.5",
     "Chart.js": "~2.1.4",
     "lodash": "~3.10.1",
     "components-font-awesome": "~4.5.0",
     "angular-poller": "~0.4.5",
     "moment": "^2.11.2",
-    "angular-toastr": "^1.7.0",
-    "pusher": "^3.0.0",
-    "angular-messages": "^1.5.3"
+    "pusher": "^3.0.0"
```

Comment:
Noox `^3.0.0` will not resolve to `4.0.0`.. It will resolve 3.x.. so at the moment that is `3.2.4`. I would rather not upgrade Pusher to a new major version with breaking changes in this release. What do you think guys? @alexnoox @cesar-tonnoir